### PR TITLE
Added documentation landing page

### PR DESCRIPTION
**Pull request by MiaAltieri**
Created documentation landing page based around Kubeflow Landing Page

### CODE REVIEW

**Review by deusebio — APPROVED** [review]
All good!

File: documentation/documentation_landing.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Charmed MongoDB Documentation 
+The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database. It is an open source, end-to-end, production ready data platform on top of cloud native technologies.
```
Comment (delgod) [inline documentation/documentation_landing.md:2]: I don't think our operator is cloud-native or juju is cloud-native.
```suggestion
The Charmed MongoDB Operator delivers automated operations management from day 0 to day 2 on the [MongoDB Community Edition](https://github.com/mongodb/mongo) document database. It is an open source, end-to-end, production-ready data platform on top of juju.
```

**Review by delgod — CHANGES_REQUESTED** [review]

**Review by deusebio — APPROVED** [review]
Looks good, just a couple of suggestion

**Review by delgod — APPROVED** [review]

File: documentation/documentation_landing.md [hunk 2]
```diff
@@ -5,18 +5,26 @@ MongoDB is a popular NoSQL database application. It stores its data with JSON-li
...
+
+# Navigation
+
+| Level | Path     | Navlink                         |
+| ----- | -------- | ------------------------------- |
+| 1 | Tutorials | [Tutorials]() |
+| 2 | getting-started | [Getting started]() |
+| 1 | how-tos-coming-soon | [How To -- Coming Soon]() |
+| 1 | explanation-coming-soon | [Explanation -- Coming Soon]() |
+| 1 | reference-coming-soon | [Reference -- Coming Soon]() |
```
[the remark this answers is not in the document]
Comment (MiaAltieri) [reply documentation/documentation_landing.md:24]: We could put I believe these links have to be links to discourse posts?

[the remark this answers is not in the document]
Comment (deusebio) [reply documentation/documentation_landing.md:24]: If this is not possible, we can of course skip this

**Pull request merged**